src/models/TranslationValue/addTranslationValueToTranslationKey/index.js

```diff
@@ -66,27 +59,27 @@ const addTranslationValueToTranslationKey = async ({
     throw new Error('This locale is not activated for this project')
   }
 
-  const valueValidation = validateValue(value)
-
-  if (!valueValidation.isValid) {
-    errors.push({
-      field: 'value',
-      message: valueValidation.error
+  // Saving key
+  try {
+    const existingTranslationValue = await TranslationValue.query().findOne({
+      translationKeyId: translationKey.id,
+      localeId: locale.id
     })
 
-    return { errors }
-  }
+    if (!value) {
+      if (existingTranslationValue) {
+        // Delete the translation value if the new value is falsy (empty or null).
+        await TranslationValue.query().deleteById(existingTranslationValue.id)
+      }
+
+      // $FlowFixMe
+      return { translationKey: null, errors }
```

Comment:
We simply don’t create a translation value if the value is falsy

Reply:
Since we decided not to return anything but errors and status when mutating, could we refactor this not to return anything ?

Reply:
we could yup